### PR TITLE
PMM-6309 replace or remove offensive terms

### PR DESCRIPTION
**Pull request by PaulJacobs-percona**
Many cannot be removed as they are technical elements (variables, commands, etc). Others live in links to docs (will be resolved with PMM-7046)

### CODE REVIEW

File: docs/js/version-select.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  // template refers to...
+  var ex = new RegExp("/?css/version-select.css$");
+  var sheet = document.querySelector('link[href$="version-select.css"]');
+
+  var ABS_BASE_URL = sheet.href.replace(ex, "");
+  var CURRENT_VERSION = ABS_BASE_URL.split("/").pop();
+
+  function makeSelect(options, selected) {
+    var select = document.createElement("select");
+//    select.classList.add("form-control");
```
Comment (atymchuk) [inline docs/js/version-select.js:12]: Redundant code

File: mkdocs.yml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 site_name: Percona Monitoring and Management
-site_url: https://www.percona.com/doc/percona-monitoring-and-management/2.x/
+#site_url: https://new.percona.com/doc/percona-monitoring-and-management/2.x/
```
Comment (atymchuk) [inline mkdocs.yml:2]: Would it work if `site_url` is commented out?

**Review by atymchuk — CHANGES_REQUESTED** [review]

File: mkdocs.yml (hunk 2 above)
Comment (PaulJacobs-percona) [reply mkdocs.yml:2]: Yes it does and means it works on both new.percona.com and www.percona.com

File: docs/js/version-select.js (hunk 1 above)
Comment (PaulJacobs-percona) [reply docs/js/version-select.js:12]: Residue from testing

**Pull request merged**